src/pages/wallets/find-wallet.tsx

```diff
@@ -138,7 +139,13 @@ const FindWalletPage = () => {
   }
 
   return (
-    <Flex as={MainArticle} direction="column" position="relative" w="full">
+    <Flex
+      as={MainArticle}
+      direction="column"
+      position="relative"
+      w="full"
+      suppressHydrationWarning
```

Comment:
I don't think it is a good idea to suppress this warning. At some point we should make the effort to fix the root problem.
```suggestion
```

Reply:
Agree here, should this be fixed in the PR open for wallet filtering? Pretty sure the hydration error is caused by the shuffle being done client side.